examples/counter/crate/src/lib.rs

```diff
@@ -28,59 +28,53 @@ impl Render for Counter {
     {
         let val = bumpalo::format!(in bump, "{}", self.val);
 
-        let increment = Node::element(
-            bump,
-            "button",
-            [Attribute {
-                name: "data-action",
-                value: "increment",
-            }],
-            [Node::text("+")],
-        );
-
-        let decrement = Node::element(
-            bump,
-            "button",
-            [Attribute {
-                name: "data-action",
-                value: "decrement",
-            }],
-            [Node::text("-")],
-        );
-
         Node::element(
             bump,
             "div",
             [],
-            [increment, Node::text(val.into_bump_str()), decrement],
+            [],
+            [
+                Node::element(
```

Comment:
Was the movement from locals to this array stylistic or required for lifetimes?

Reply:
Stylistic.